LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionFactory.cs

```diff
@@ -37,23 +37,41 @@ public static bool TryGetRegion(LoRaRegion value, out Region region)
         }
 

```

Comment:
The region situation is a bit messy atm. I have another PR ready that revamp the region management.

Reply:
This method is like any other bool TryXXX(out object outParameter).
If we cannot resolve we return false and set outParameter to null.

This method was created because the current implementation sets the result into a static variable making it unsafe for tests.